lib/fastlane/plugin/firebase_app_distribution/client/firebase_app_distribution_api_client.rb

```diff
@@ -198,33 +207,54 @@ def get_udids(app_id)
 
       private
 
-      def v1_apps_url(app_id)
+      def project_number(app_id)
+        app_id.split(':')[1]
+      end
+
+      def v1alpha_apps_url(app_id)
```

Comment:
I think all these methods should take resource names (e.g. `app_name` and `release_name`) as parameters instead of IDs.

Reply:
Done!